addition.c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+int main()
+{
+    int a, b, sum;
+
+    printf("Enter two numbers to add");
+    scanf("%d%d", &a, &b);
+
+    sum = a + b;
+
```

Comment:
Do we really need the sum here? Can we optimize it? And when should we declare a variable?

Reply:
No, it is not necessary. This variable optimized as firstnum + secondnum